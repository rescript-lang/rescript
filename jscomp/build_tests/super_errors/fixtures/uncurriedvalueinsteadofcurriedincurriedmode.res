module Foo: {
  let adducc: (int, int) => int
} = {
  let adducc = (. a, b) => a + b
}
