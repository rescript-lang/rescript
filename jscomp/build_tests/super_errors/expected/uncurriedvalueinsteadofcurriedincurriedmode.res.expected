
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/uncurriedvalueinsteadofcurriedincurriedmode.res[0m:[2m3:5-5:1[0m

  1 [2mâ”‚[0m module Foo: {
  2 [2mâ”‚[0m   let adducc: (int, int) => int
  [1;31m3[0m [2mâ”‚[0m } = [1;31m{[0m
  [1;31m4[0m [2mâ”‚[0m [1;31m  let adducc = (. a, b) => a + b[0m
  [1;31m5[0m [2mâ”‚[0m [1;31m}[0m
  6 [2mâ”‚[0m 

  Signature mismatch:
  Modules do not match:
    {
  let adducc: (. int, int) => int
}
  is not included in
    {
  let adducc: (int, int) => int
}
  Values do not match:
    let adducc: (. int, int) => int (uncurried)
  is not included in
    let adducc: (int, int) => int (curried)
  [36m/.../fixtures/uncurriedvalueinsteadofcurriedincurriedmode.res[0m:[2m2:3-31[0m:
    Expected declaration
  [36m/.../fixtures/uncurriedvalueinsteadofcurriedincurriedmode.res[0m:[2m4:7-12[0m:
    Actual declaration